projects/storefrontlib/src/cms-components/cart/cart-coupon/cart-coupon.component.html

```diff
@@ -88,4 +34,30 @@
     [isReadOnly]="false"
   >
   </cx-applied-coupons>
+
+  <ng-container *ngIf="applicableCoupons && applicableCoupons.length > 0">
+    <div class="cx-available-coupon">
+      <div class="title">
+        {{ 'voucher.availableCoupons' | cxTranslate }}
+      </div>
+      <div class="message">
+        {{ 'voucher.availableCouponsLabel' | cxTranslate }}
+      </div>
+      <div class="scroll">
+        <div
+          class="coupons card "
+          *ngFor="let coupon of applicableCoupons"
+          (click)="applyCustomerCoupon(coupon.couponId)"
```

Comment:
`<div>` shouldn't have `click()` event. please move this logic to the button

Reply:
fix